Shared/Network/NetworkManager.swift

```diff
@@ -95,10 +95,10 @@ struct NetworkManager {
                         menu.forEach { item in
                             let items = item.items.compactMap { $0 }
                             items.forEach { menuItem in
-                                allMenuItems.append(Menu.Item(name: menuItem.item, healthy: menuItem.healthy))
+                                allMenuItems.append(Menu.Item(name: menuItem.item, healthy: menuItem.healthy, favorite: menuItem.favorite))
                             }
                             categoryToMenu[item.category] = items.map { itemForEvent in
-                                Menu.Item(name: itemForEvent.item, healthy: itemForEvent.healthy)
+                                Menu.Item(name: itemForEvent.item, healthy: itemForEvent.healthy, favorite: itemForEvent.favorite)
```

Comment:
This and lines 98 and 177 are pretty long. You should break them up onto multiple lines. Also, if you don't have SwiftLint installed, you should consider getting that set up. Let me know if you'd need help with that.